manage.py

```diff
@@ -32,6 +32,37 @@
 manager.add_command('runserver', Server(use_debugger=True, use_reloader=True))
 
 
+def check_itemized_queues(schedule):
```

Comment:
Likely not, but I left it in for the time being since I'm sure we'll be revisiting this soon.  The calls to it have been removed, though.